src/__tests__/helpers/events.js

```diff
@@ -20,5 +19,6 @@ export function createMoveTouchEventObject({ x = 0, y = 0, includeTouches = true
 export function createMouseEventObject({ x = 0, y = 0 }) {
   return {
     ...createClientXYObject(x, y),
+    preventDefault: jest.fn(),
```

Comment:
- [ ] maybe make this an arg you can pass in so we can test if `preventDefault` and `stopPropagation` were fired?